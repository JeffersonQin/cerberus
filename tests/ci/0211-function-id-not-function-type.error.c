typedef int fun(void);
fun f;
fun f {}
int main() {}
