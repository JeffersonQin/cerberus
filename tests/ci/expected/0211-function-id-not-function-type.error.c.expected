ci/0211-function-id-not-function-type.error.c:3:5: error: constraint violation: function identifier must have a function type
fun f {}
    ^ 
ยง6.9.1#2: 
2   The identifier declared in a function definition (which is the name of the function) shall
    have a function type, as specified by the declarator portion of the function definition.162)
